bin/brew

```diff
@@ -1,14 +1,27 @@
 #!/bin/bash
 
-chdir() {
-  cd "$@" >/dev/null || {
-    echo "Error: failed to cd to " "$@" "!" >&2
+odie() {
+    if [[ -t 2 ]]
+    then
+        echo -ne "\033[4;31mError\033[0m: " >&2
```

Comment:
Thanks for pointing that out! I didn't even realize this is the case. Then I'm all for keeping it like that to stay consistent. (And if it bothers me enough, I can rectify this for both in a follow-up PR.)